fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/CachedStatisticStorage.java

```diff
@@ -113,6 +113,9 @@ public List<ColumnStatistic> getColumnStatistics(Table table, List<String> colum
 
     @Override
     public void expireColumnStatistics(Table table, List<String> columns) {
+        if (columns == null) {
```

Comment:
Why columns could be null in here?

Reply:
At present, it is not possible to be NULL. I just added a check here. I will change it to Preconditions.checkNotNull.

Reply:
The column information in basicMeta is added later, there may be EditLog playback that is not recorded, then the column may be Null, here is for compatibility consideration